File: Test_task/Document .Net (5.2.2.16)/Code samples/CSharp/12. Product Installation/02 .Net Core/Sample.cs
using SautinSoft.Document;

namespace Sample
{
    class Sample
    {
        static void Main(string[] args)
        {
            ExampleHelloWorld();
        }
		
        /// <summary>
        /// Create file with Hello World.
        /// </summary>
        /// <remarks>
        /// Details: https://sautinsoft.com/products/document/help/net/developer-guide/using-in-netcore.php
        /// </remarks>
        static void ExampleHelloWorld()
        {
            DocumentCore dc = new DocumentCore();
            dc.Content.End.Insert("Hello World!", new CharacterFormat() { FontName = "Verdana", Size = 65.5d,FontColor = Color.DarkBlue });
			
            // Save a document to a file in PDF format.
            string filePath = @"Result.pdf";
            dc.Save(filePath);

            // Open the result for demonstration purposes.
            System.Diagnostics.Process.Start(new System.Diagnostics.ProcessStartInfo(filePath) { UseShellExecute = true });
        }
    }
}

File: Test_task/Document .Net (5.2.2.16)/Code samples/VB.Net/12. Product Installation/02 .Net Core/Sample.vb
Imports System
Imports System.IO
Imports SautinSoft.Document

Module Sample
    Sub Main()
        ExampleHelloWorld()
    End Sub

    ''' <summary>
    ''' Create file with Hello World.
    ''' </summary>
    ''' <remarks>
    ''' Details: https://sautinsoft.com/products/document/help/net/developer-guide/using-in-netcore.php
    ''' </remarks>
    Sub ExampleHelloWorld()
        Dim dc As New DocumentCore()
        dc.Content.End.Insert("Hello World!", New CharacterFormat() With {
                .FontName = "Verdana",
                .Size = 65.5R,
                .FontColor = Color.DarkBlue
            })

        ' Save a document to a file in PDF format.
        Dim filePath As String = "Result.pdf"
        dc.Save(filePath)

        ' Open the result for demonstration purposes.
        System.Diagnostics.Process.Start(New System.Diagnostics.ProcessStartInfo(filePath) With {.UseShellExecute = True})
    End Sub
End Module